src/Add_remove.js

```diff
@@ -0,0 +1,36 @@
+const TodoLoad = (todos) => {
+  todos = JSON.parse(localStorage.getItem('todos')) || [];
+  for (let i = 0; i < todos.length; i += 1) {
+    todos[i].index = i + 1;
+  }
+  return todos;
+};
+
+class Todo {
+  constructor(description, index = 0, completed = false) {
+    this.index = index;
+    this.description = description;
+    this.completed = completed;
+  }
+
```

Comment:
As much as you can better use class to encapsulate your code remember the principles of **DRY**